ClickHouse/clickhouse-kafka-connect
# add manifest.json

Add `manifest.json` to comply with Confluent Hub requirements. see https://docs.confluent.io/kafka-connectors/self-managed/confluent-hub/component-archive.html#c-hub-component-archive-specification

How to test:
`./gradlew createConfluentArchive; cd build/confluent`, check the archive content
